src/main/java/io/github/thebusybiscuit/slimefun4/utils/tags/SlimefunTag.java

```diff
@@ -260,9 +260,24 @@ public enum SlimefunTag implements Tag<Material> {
     GRAVITY_AFFECTED_BLOCKS,
 
     /**
-     * All wool carpets
+     * All wool carpets.
      */
-    WOOL_CARPETS;
+    WOOL_CARPETS,
+
+    /**
+     * All storage blocks.
+     */
+    AUTO_CRAFTER_SUPPORTED_STORAGE,
+
+    /**
+     * All blocks with an inventory.
+     */
+    CARGO_SUPPORTED_INVENTORIES,
```

Comment:
These comments need to be updated.
Also some consistency in naming would probably be better, e.g. "XYZ_SUPPORTED_STORAGE_BLOCKS"